### PR TITLE
js done

### PR DESCRIPTION
**Pull request by PedroBarge**
JS done. pls do CSS

### CODE REVIEW

**Review by DaveLRB — APPROVED** [review]
logic looks sound not overcomplicated, will use elements created to improve the current design. Good job

**Review by FranciscoSilvaMgLPT — APPROVED** [review]

**Comment by FranciscoSilvaMgLPT** [comment]
esta lindo!

File: .DS_Store
Comment (201flaviosilva) [inline .DS_Store:1]: Remove this file;
Update the `.gitignore` file to this: https://www.toptal.com/developers/gitignore/api/macos,node,visualstudiocode

**Comment by PedroBarge** [comment]
done

**Review by DaveLRB — APPROVED** [review]
Good

**Pull request merged**